plugins/CorePluginsAdmin/templates/macros.twig

```diff
@@ -1,10 +1,12 @@
-{% macro tablePluginUpdates(pluginsHavingUpdate, nonce, isTheme) %}
+
+{% macro tablePluginUpdates(pluginsHavingUpdate, updateNonce, isMultiServerEnvironment) %}
+    {% import '@Marketplace/macros.twig' as marketplaceMacro %}
 
     <div class='entityContainer'>
         <table class="dataTable entityTable">
             <thead>
             <tr>
-                <th>{% if isTheme %}{{ 'CorePluginsAdmin_Theme'|translate }}{% else %}{{ 'General_Plugin'|translate }}{% endif %}</th>
```

Comment:
This was never working before, I mean isTheme was buggy there as another variable was passed there